pilot/pkg/model/service.go

```diff
@@ -558,6 +558,11 @@ func BuildSubsetKey(direction TrafficDirection, subsetName string, hostname host
 	return string(direction) + "|" + strconv.Itoa(port) + "|" + subsetName + "|" + string(hostname)
 }
 
+// BuildInboundSubsetKey generates a unique string referencing service instances with port.
+func BuildInboundSubsetKey(subsetName string, hostname host.Name, port int) string {
```

Comment:
can we remove the subsetName and hostname

Reply:
We can remove. Just kept it incase we need it in future. But we can add back when we need it. I will remove